dev/com.ibm.ws.artifact.zip/src/com/ibm/ws/artifact/zip/internal/ZipFileContainerFactory.java

```diff
@@ -352,7 +352,10 @@ private static boolean isZip(ArtifactEntry artifactEntry) {
             try {
                 ZipEntry entry = zipInputStream.getNextEntry();
                 if ( entry == null ) {
-                    Tr.error(tc, "bad.zip.data", getPhysicalPath(artifactEntry));
+                    JavaArchive javaArchive = new JavaArchive(getPhysicalPath(artifactEntry));
```

Comment:
Does the JavaArchive need to be closed?

Reply:
Well, not here.   The JavaArchive.isValid() method opens the archive file in a try with resources.   So it should be closed automagically.

Reply:
Well, not here.   The JavaArchive.isValid() method opens the archive file in a try with resources.   So it should be closed automagically.